nixos/modules/programs/envision.nix

```diff
@@ -19,6 +19,12 @@ in
       openFirewall = lib.mkEnableOption "the default ports in the firewall for the WiVRn server" // {
         default = true;
       };
+
+      otherPackages = {
+        xr-hardware = lib.mkPackageOption pkgs "xr-hardware" {};
+
+        android-udev-rules = lib.mkPackageOption pkgs "android-udev-rules" {};
+      };
```

Comment:
We might be able to omit this. I don't think many modules expose auxiliary packages like this.

Reply:
Fair enough. I'll make it non-configurable.